src/components/chips/chips-theme.scss

```diff
@@ -1,4 +1,4 @@
-md-chips.md-THEME_NAME-theme {
+md-chips.md-THEME_NAME-theme, data-md-chips.md-THEME_NAME-theme {
```

Comment:
I don't like that. This would require a selector for `x-` as well.

We should either apply a class, to identify the chips directive.